src/tally.F90

```diff
@@ -441,6 +441,69 @@ subroutine score_general_ce(p, t, start_index, filter_index, i_nuclide, &
         end if
 
 
+      case (SCORE_PROMPT_NU_FISSION)
```

Comment:
I think it could be simplified such that the total/prompt/delayed are all handled in the same method via an integer argument to the routine. I'll look into this.